packages/dotnet/ReadMe/Metrics.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Configuration;
-using Readme.HarJsonObjectModels;
-using Readme.HarJsonTranslationLogics;
+using ReadMe.HarJsonObjectModels;
+using ReadMe.HarJsonTranslationLogics;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Readme
+namespace ReadMe
```

Comment:
Hmm yeah I actually think it may be. It seems as though NuGet packages are case insensitive, you can do any of these and it installs [this](https://www.nuget.org/packages/ReadMe.Metrics/):

```sh
dotnet add package ReadMe.Metrics
dotnet add package readme.metrics
dotnet add package ReAdMe.MeTrIcS
```

But the namespace that gets brought in to your actual code does need to match exactly what is listed in our code. 

So we'll have to do a major version bump also have to update [this](https://docs.readme.com/docs/net-setup#aspnet-core-integration) when this gets merged. Is that content synced from anywhere? Should it be synced in from this repo?

Reply:
Yeah if you update that in the `docs/` repo it'll get synced there when this is merged into `main`.

Reply:
Updated here: 689cfb597188bc75cbf07b4e57702b9ef7d07600